### PR TITLE
Tianheng dev

### PR DESCRIPTION
**Pull request by tianhengzhou**
Tianheng Zhou' HW 1

### CODE REVIEW

**Comment by tianhengzhou** [comment]
Tianheng Zhou' HW 1. There are some hidden files, they come with the Webstorm I think. Please ignore them.

**Comment by techbow** [comment]
What are these hidden files, if you use source tree, you can remove them easily from your PR

File: Tianheng_HW1/css/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    margin-left: auto;
+    margin-right: auto;
+}
+aside{
+    float: left;
+    width:43%;
+    margin-left: auto;
+    margin-right: auto;
+}
+#MainFeature{
```
Comment (techbow) [inline Tianheng_HW1/css/stylesheet.css]: usually, by convention, we just use class for styles purpose and use id for javascript to hook up element

File: Tianheng_HW1/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+<head lang="en">
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/stylesheet.css"/>
+    <title>Orange Udacity Mug</title>
+</head>
+<body>
+<header>Orange Udacity Mug</header>
+<aside role="complementary">
+    <div id="image">
+    <img src="mug.png" alt="mug picture"/>
```
Comment (techbow) [inline Tianheng_HW1/index.html:12]: indention is important

File: Tianheng_HW1/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    </div>
+    <div>Ad aliquam amet animi cumque eius eligendi, et excepturi explicabo, hic ipsam iste labore maiores officiis
+        placeat quos recusandae repellendus sit velit? Animi architecto at distinctio ipsam iure non saepe.
+    </div>
+    <div>Ad aliquam amet animi cumque eius eligendi, et excepturi explicabo, hic ipsam iste labore maiores officiis
+        placeat quos recusandae repellendus sit velit? Animi architecto at distinctio ipsam iure non saepe.
+    </div>
+    <div>Ad aliquam amet animi cumque eius eligendi, et excepturi explicabo, hic ipsam iste labore maiores officiis
+        placeat quos recusandae repellendus sit velit? Animi architecto at distinctio ipsam iure non saepe.
+    </div>
```
Comment (techbow) [inline Tianheng_HW1/index.html:67]: it is not a good idea to use so many div just for display block purpose

File: Tianheng_HW1/css/stylesheet.css (hunk 1 above)
Comment (tianhengzhou) [inline Tianheng_HW1/css/stylesheet.css]: Ok. That's a really important suggestion. Good to know. I'll modify the code.

File: Tianheng_HW1/index.html (hunk 3 above)
Comment (tianhengzhou) [inline Tianheng_HW1/index.html:67]: Ok, I'll change it. BTW this is actually automatically generate by emmet which is an extension for HTML and CSS fast programming.

**Comment by tianhengzhou** [comment]
There is some file like .idea/.name, if you go to file changed tab on the top of this page, you'll see them. But I think they're Ok. It just some files automatically generates by Webstorm IDE.

**Pull request merged**